src/Shutdown/ShutdownStrategyInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Bugsnag\Shutdown;
+
+use Bugsnag\Client;
+
+/**
+ * Interface ShutdownStrategyInterface.
```

Comment:
Worth expanding the docs here to describe the purpose of the interface and expected reasons for implementing it?